golded3/gepost.cpp

```diff
@@ -389,6 +389,7 @@ static void MakeMsg3(int& mode, GMsg* msg)
             A->SaveMsg(GMSG_NEW|GMSG_NOLSTUPD, msg);
             A->Close();
             AA->Open();
+            AA->RandomizeData(mode);
```

Comment:
Does this solve the following bug?
= GOLDED (2:5020/1042) ========================================================
 From : Tommi Koivula                       2:221/1         Sun 28 Nov 21 10:01
 To   : All
 Subj : areafile husky
===============================================================================
* Originally in golded
* Crossposted in fidosoft.husky

Hi All,

I use GoldED with -W to create echolist to be used elsewhere.

 "-W  Write a GOLDAREA.INC file with AREADEF's of all mail areas."

Some groups were shown as "0", not as "X" or "F" or as they should. I
investigated this a bit and found out that if "-TooOld" is before "-g F" in
echoarea line, GoldED does not read the group correctly.

Problem solved by reordering parameters in fidoconfig.

'Tommi

---
 * Origin: rbb.fidonet.fi (2:221/1)

Reply:
OK. When you have time, please, fix the bug from Tommy's message too. It's not a first priority bug, so you are to decide what to do first.

Reply:
I'll take a look on it from his description looks simple.